code401Challenges/src/main/java/code401Challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,77 @@
+package code401Challenges.linkedlist;
+
+import java.util.ArrayList;
+
+public class LinkedList {
+    Node head;
+
+    LinkedList(){
```

Comment:
You should also get in the habit of specifying whether a constructor is public, protected or private